config/routes.rb

```diff
@@ -18,32 +15,34 @@
   resources :orders
 
   namespace :admin do
+    get 'dashboard/index'
     get '', to: 'dashboard#index', as: '/'
     get 'affiliate/:id/show_affiliate_workers',
         to: 'affiliates#show_affiliate_workers',
         as: :show_affiliate_workers
     resources :users
     resources :affiliates
-    resources :taxes
+    resources :taxes do
+      get 'delete',
+          to: 'taxes#delete',
```

Comment:
[Could not find delete action](https://github.com/unrooty/taxi_project/blob/b194f3f08c02cbdc9d3356e420d5c6879159c1fb/app/controllers/admin/taxes_controller.rb) in `taxes_controller`, maybe you meant `:destroy` (already added by Rails CRUD model for routing resources)?